benchmarks/benchmark.js

```diff
@@ -289,6 +290,16 @@ if (process.env.PORT) {
     })
   }
 
+  experiments['undici - fetch (with diagnostics channel)'] = () => {
+    const onMessage = () => {}
+    diagnosticsChannel.subscribe('undici:fetch:asyncEnd', onMessage)
+    return makeParallelRequests(resolve => {
+      fetch(dest.url).then(res => {
+        res.body.pipeTo(new WritableStream({ write () {}, close () { resolve() } }))
+      }).catch(console.log)
+    }).finally(() => { diagnosticsChannel.unsubscribe('undici:fetch:asyncEnd', onMessage) })
+  }
```

Comment:
Move this to another file, it's not benchmarking what you think it's benchmarking.

Reply:
@mcollina, since it is not, is it needed? To my understanding tracing channel should not affect performance much when it has subscribers and not at all when it does not have subscribers.

Reply:
Let's remove it

Reply:
It has been removed 👍🏼